container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend.go

```diff
@@ -473,6 +473,18 @@ func (backend *DockerKurtosisBackend) DestroyLogsCollectorForEnclave(ctx context
 	return nil
 }
 
+func (backend *DockerKurtosisBackend) DestroyLogsStorage(ctx context.Context) error {
+	logsStorageAttrs, err := backend.objAttrsProvider.ForLogsStorageVolume()
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred attempting to destroy logs storage.")
```

Comment:
nit: same error msg from both calls, you could be more specific here.

Reply:
yep good callout.